README.md

```diff
@@ -60,8 +59,6 @@
 
```

Comment:
- [ ] I was not able to inline in your `index.css` file but Thank you for your work on the project so far. I noticed that the design is slightly different from the template we were given. While I appreciate your creativity and willingness to explore different design options, it's important that the final product matches the template closely to maintain consistency and ensure the project is user-friendly :wink: Would you be willing to adjust the design to align more closely with the template? I'm happy to provide to give some guidance and support if needed. Thank you again for your hard work, kindly fix this :point_down: 

Your design 
![Screenshot from 2023-04-24 16-09-55](https://user-images.githubusercontent.com/88894525/234024685-8968e4fb-62d0-49b6-8416-d2af21d0deb8.png)

Figma Design
![Screenshot from 2023-04-24 16-05-39](https://user-images.githubusercontent.com/88894525/234024747-eeed4f2b-bbef-44e8-982f-30c1d81c0ff1.png)